src/Compilers/Core/Portable/Symbols/Attributes/CommonParameterWellKnownAttributeData.cs

```diff
@@ -50,6 +50,8 @@ public bool HasInAttribute
         }
         #endregion
 
+        // PROTOTYPE: RefReadOnlyAttribute
```

Comment:
>// PROTOTYPE: RefReadOnlyAttribute [](http://example.com/codeflow?start=8&length=34)

Are we planning to allow explicit application of this attribute to parameters in source? If not, then there is nothing to record in this class. BTW, if we decide to record presence of the attribute in source, we probably should do that for C# only (in ```ParameterWellKnownAttributeData```), and, therefore, this class shouldn't be modified.